website/src/pages/3million.tsx

```
@@ -0,0 +1,86 @@
+          name="twitter:description"
+          content="Celebrate 3,000,000 downloads of Podman Desktop! Thank you to everyone who provides feedback and helps us improve."
+        />
+        <meta name="twitter:image" content={`https://podman-desktop.io${celebrationImageUrl}`} />
+        <meta name="twitter:image:alt" content="Podman Desktop celebrates 3,000,000 downloads" />
+
+        <link rel="canonical" href="https://podman-desktop.io/3million" />
+
+        <script
+          type="module"
+          crossOrigin="anonymous"
+          src="https://unpkg.com/@splinetool/viewer@1.10.56/build/spline-viewer.js"></script>
```

Biome (2.1.2)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Safe fix: Use a self-closing element instead.


(lint/style/useSelfClosingElements)